src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-weight: 900;
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-weight: 300;
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.large__bar {
+  display: flex;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.large__icon {
+  display: block;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 45px;
+  left: 34px;
+}
+
+.input__large {
+  display: block;
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: inherit;
+}
+
+.input__large:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input__large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  outline: none;
+}
+
+.input__large:focus::placeholder {
+  font-weight: 300;
+}
+
+.small__bar {
+  display: flex;
+  width: 100%;
+}
+
+.small__icon {
+  display: inline-block;
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 125px;
+  left: 21px;
+}
+
+.input__small {
```

Comment:
Good job) Let's improve a little bit)

You have a lot of similar styles for both inputs, use styles for big input as base, and then override necessary properties for small input

[CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md ) p2